security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/AbstractTapChangerTapPositionAction.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.action;
+
+import com.powsybl.iidm.network.ThreeWindingsTransformer;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * An action modifying the tap position of a two or three windings transformer
+ *
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ * @author Anne Tilloy <anne.tilloy@rte-france.com>
+ */
+public abstract class AbstractTapChangerTapPositionAction extends AbstractAction {
+
+    private final String transformerId;
+    private final int tapPosition;
+    private final boolean relativeValue; // true if relative value chosen, false if absolute value
+    private final ThreeWindingsTransformer.Side side;
```

Comment:
Maybe something to solve later, but also for a 2 windings transformer we use the enum of a 3 windings.